### PR TITLE
Removes dtags from release notes

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### Description

Dtags in release notes have been a constant source of pain. This PR removes all dtags and consecutive blank likes from release notes in this repo.

### Check List

- [ ] Spell Check
- [x] Local build
- [x] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: chef_master/source/release_notes_chefdk.rst [hunk 1]
```diff
@@ -951,7 +943,6 @@ What's New in 1.0
 Version 1.0!
 -----------------------------------------------------
 
-We're recognizing ChefDK's continued stability with the honor of a 1.0 tag. There
```
Comment (tas50) [inline chef_master/source/release_notes_chefdk.rst:954]: We probably want this

Comment (kagarmoe) [reply chef_master/source/release_notes_chefdk.rst:954]:
```suggestion
We're recognizing ChefDK's continued stability with the honor of a 1.0 tag. There
```

Comment (kagarmoe) [reply chef_master/source/release_notes_chefdk.rst:954]: I might need you to hit the "commit suggestion" button, because it isn't showing for me.

Comment (tas50) [reply chef_master/source/release_notes_chefdk.rst:954]: oh yeah no DCO this is a perfect place for it

Comment (tas50) [reply chef_master/source/release_notes_chefdk.rst:954]: Nope you can't add back a line with suggestions. There's a few deleted lines that look like accidents

Comment (kagarmoe) [reply chef_master/source/release_notes_chefdk.rst:954]: Sadly, I don't think we can use it on deleted lines.

File: chef_master/source/release_notes_server.rst [hunk 2]
```diff
@@ -437,20 +435,15 @@ About Chef Licenses
 
 All Chef products have a license that governs the entire product and includes links to license files for any third-party software included in Chef packages. The ``/opt/<PRODUCT-NAME>/LICENSES`` directory contains individual copies of all referenced licenses.
 
-.. warning:: .. tag chef_license_note_current
```
Comment (tas50) [inline chef_master/source/release_notes_server.rst:440]: Do we want the warning here?

File: chef_master/source/release_notes_chef_automate.rst [hunk 3]
```diff
@@ -127,7 +121,6 @@ Resolved Issues
 * Stale Elasticsearch lock files can be cleaned after a crash with ``automate-ctl delete-elasticsearch-lock --stale-lock-only``
 * Fixed CA certificate path location on SuSE
 * The data collector API now returns a 503 when Automate is in maintenance mode
-* The Elasticsearch API now returns 400 if sent a HTML script tag to prevent XSS attacks
```
Comment (tas50) [inline chef_master/source/release_notes_chef_automate.rst]: Was this intentional?

File: chef_master/source/release_notes.rst [hunk 4]
```diff
@@ -4332,12 +4112,8 @@ The ksh resource has the following properties:
 
    An array of paths to use when searching for a command. These paths are not added to the command's environment $PATH. The default value uses the system path.
 
-   .. warning:: .. tag resources_common_resource_execute_attribute_path
```
Comment (kagarmoe) [inline chef_master/source/release_notes.rst:4335]: Keep this warning

File: chef_master/source/release_notes.rst [hunk 5]
```diff
@@ -6044,20 +5615,14 @@ This argument has the following options:
 ``-k``, ``--prevent-keygen``
    Create a user without a public key. This key may be managed later by using the ``knife user key`` subcommands.
 
-   .. note:: .. tag notes_knife_prevent_keygen
```
Comment (kagarmoe) [inline chef_master/source/release_notes.rst:6047]: Keep this note

File: chef_master/source/release_notes.rst [hunk 6]
```diff
@@ -6071,12 +5636,8 @@ This argument has the following options:
...
```
Comment (kagarmoe) [inline chef_master/source/release_notes.rst:6074]: Keep note

File: chef_master/source/release_notes.rst [hunk 7]
```diff
@@ -8104,17 +7458,13 @@ Examples
...
-.. end_tag
-
 New client.rb Settings
 -----------------------------------------------------
 The following client.rb settings are new:
 
 ``chef_gem_compile_time``
    Controls the phase during which a gem is installed on a node. Set to ``true`` to install a gem while the resource collection is being built (the "compile phase"). Set to ``false`` to install a gem while the chef-client is configuring the node (the "converge phase"). Recommended value: ``false``.
 
-   .. note:: .. tag resource_package_chef_gem_attribute_compile_time
```
Comment (kagarmoe) [inline chef_master/source/release_notes.rst:8116]: keep note

File: chef_master/source/release_notes.rst [hunk 8]
```diff
@@ -6159,25 +5717,14 @@ The following items are new for chef-client 12.2 and/or are changes from previou
...
 
-.. end_tag
-
-.. tag resource_dsc_resource_summary
 
 The **dsc_resource** resource allows any DSC resource to be used in a Chef recipe, as well as any custom resources that have been added to your Windows PowerShell environment. Microsoft `frequently adds new resources <https://github.com/powershell/DscResources>`_ to the DSC resource collection.
 
-.. end_tag
-
-.. warning:: .. tag resource_dsc_resource_requirements
```
Comment (kagarmoe) [inline chef_master/source/release_notes.rst:6180]: keep warning

File: chef_master/source/release_notes_chef_automate.rst (hunk 3 above)
Comment (kagarmoe) [inline chef_master/source/release_notes_chef_automate.rst]: Keep this line

File: chef_master/source/release_notes_server.rst [hunk 9]
```diff
@@ -2762,18 +2582,11 @@ The ``install`` subcommand downloads packages from https://packages.chef.io/ by
 
           $ sudo chef-manage-ctl reconfigure
 
-       .. note:: .. tag chef_license_reconfigure_manage
```
Comment (kagarmoe) [inline chef_master/source/release_notes_server.rst:2765]: keep note

**Review by IanMadd — APPROVED** [review]
everything looks good.

**Pull request merged**